l10n_it_fatturapa_out/views/attachment_view.xml

```diff
@@ -86,12 +94,10 @@
             <field name="view_type">form</field>
             <field name="view_mode">tree,form</field>
             <field name="view_id" ref="view_fatturapa_out_attachment_tree"/>
+            <field name="context">{'search_default_ready': True, 'search_default_sender_error': True, 'search_default_rejected': True, }</field>
         </record>
 
-        <menuitem action="action_fatturapa_attachment"
-            parent="l10n_it_fatturapa.menu_fattura_pa_receivables"
-            name="E-invoice Export Files"
-            id="fatturapa_attachment_menu"/>
+        <menuitem action="action_fatturapa_attachment" parent="l10n_it_fatturapa.menu_fattura_pa_receivables" name="E-invoice Export Files" id="fatturapa_attachment_menu"/>
```

Comment:
Qui, e in diversi altri punti, hai modificato la formattazione.
A me personalmente sembra più chiara quella precedente.